README.md

```diff
@@ -7,3 +7,20 @@ This is a suitcase subpackage for writing a particular file format.
 ```
 pip install suitcase-msgpack
 ```
+
+## Quick Start
+
+```
+import suitcase.msgpack
+docs = db[-1].documents(fill=True)
+suitcase.msgpack.export(docs, 'my_exported_files/')
+```
+The exported file will be saved as
+`my_exported_files/8fe81a89-8103-487a-85d2-23c70af96f16-primary.msgpack`.
+The file prefix `c416e70a-eb99-4964-bfc0-64e291313cce` is the unique ID of the
```

Comment:
Should not it be `8fe81a89-8103-487a-85d2-23c70af96f16` here? Where does `c416e70a-eb99-4964-bfc0-64e291313cce` come from?